nbstripout/_utils.py

```diff
@@ -1,6 +1,10 @@
 import sys
 
-__all__ = ["pop_recursive", "strip_output"]
+__all__ = ["pop_recursive", "strip_output", "NBStripoutError"]
+
+
+class NBStripoutError(Exception):
```

Comment:
Do we want to call this e.g. `MetadataError`?

Reply:
Renamed as you suggested